docker/dev/Dockerfile.torch.ngc

```diff
@@ -0,0 +1,21 @@
+ARG BASEIMAGE=nvcr.io/nvidia/pytorch:21.11-py3
+FROM ${BASEIMAGE}
+
+COPY docker/scripts /install/scripts
+
+RUN mkdir -p /usr/local/TensorRT && \
+    ln -s /usr/include/x86_64-linux-gnu/ /usr/local/TensorRT/include && \
```

Comment:
looks weird ...🤔

Reply:
NGC's TensorRT library is installed in `/lib/x86_64-linux-gnu/`. But our build scripts use `/usr/local/TensorRT` as the install path. It might be a little hack, but I think it's clean.